app/views/contributions/_contribution_list.erb

```diff
@@ -14,14 +14,14 @@
     <span class="deadmark">
 
     </span>
-    <%= @contribution.content %>
+    <%= @contributions.content %>
```

Comment:
I'd solve it but I can't see the problem. @contributions variable is defined in contributions controller